Sources/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -597,6 +692,8 @@ class MessageUpdater: Worker {
             attachmentDTO.localState = .pendingUpload
         }, completion: completion)
     }
+    
+    // MARK: -
```

Comment:
Some empty doce here:
```suggestion
    // MARK: - ???
```